handlers/deploy.go

```diff
@@ -118,16 +116,19 @@ func MakeDeployHandler(functionNamespace string, clientset *kubernetes.Clientset
 
 func makeDeploymentSpec(request requests.CreateFunctionRequest, existingSecrets map[string]*apiv1.Secret, config *DeployHandlerConfig) (*v1beta1.Deployment, error) {
 	envVars := buildEnvVars(&request)
-	path := filepath.Join(os.TempDir(), ".lock")
 	probe := &apiv1.Probe{
 		Handler: apiv1.Handler{
-			Exec: &apiv1.ExecAction{
-				Command: []string{"cat", path},
+			HTTPGet: &apiv1.HTTPGetAction{
```

Comment:
This can't be turned on by default, it needs to be optional.

Reply:
Oh, I had thought based on our off-line conversation this wasn't the case.  Easy to make optional